docs/docs-components/QualityChecklist.js

```diff
@@ -60,6 +68,79 @@ export default function QualityChecklist({ component }: Props): Node {
                 </Table.Row>
               );
             })}
+            {data?.status.knownIssues ? (
+              <Table.RowExpandable
+                accessibilityExpandLabel="Expand"
+                accessibilityCollapseLabel="Collapse"
+                id="known_issues"
+                expandedContents={
+                  <List labelDisplay="hidden" label="Known issues for component" type="unordered">
+                    {data?.status.knownIssues.map(
+                      ({ title, description, codesandboxUrl, internalDocUrl }, idx) => (
+                        <List.Item
+                          key={`${idx + title}`}
+                          text={
+                            <Text inline>
+                              <Text weight="bold" inline>
+                                {title}.{' '}
```

Comment:
We might not need a period here

Reply:
Like the linClamp example seems little weird

Reply:
What do you mean with weird?